lighthouse-cli/test/cli/index-test.js

```diff
@@ -82,8 +82,46 @@ describe('CLI Tests', function() {
   });
 
   describe('print-config', () => {
+    describe('fraggle rock', () => {
+      it('should print the default config and exit immediately after', () => {
```

Comment:
> That condition can be added to a more specific test though so I could go either way.

Maybe could do `--only-audits charset` instead of metrics in the test below this? That should require `MainDocumentContent` which depends on `DevtoolsLog`.